--- src/corelib/global/qlibraryinfo.cpp	2011-12-07 22:06:03.000000000 -0700
+++ src/corelib/global/qlibraryinfo.cpp	2011-12-15 10:59:40.416624000 -0700
@@ -476,7 +476,7 @@
             // we make any other path absolute to the prefix directory
             baseDir = location(PrefixPath);
         }
-        ret = QDir::cleanPath(baseDir + QLatin1Char('/') + ret);
+        ret = QDir::toNativeSeparators(QDir::cleanPath(baseDir + QLatin1Char('/') + ret));
     }
     return ret;
 }
