--- src/corelib/global/qlibraryinfo.cpp	2009-12-23 09:08:31.745250000 -0700
+++ src/corelib/global/qlibraryinfo.cpp	2009-12-22 23:43:29.142625000 -0700
@@ -450,7 +450,7 @@
 #endif
         } else {
             // we make any other path absolute to the prefix directory
-            return QDir(location(PrefixPath)).absoluteFilePath(ret);
+            return QDir::toNativeSeparators(QDir(location(PrefixPath)).absoluteFilePath(ret));
         }
     }
     return ret;
