diff --git a/src/gui/styles/qwindowsxpstyle.cpp b/src/gui/styles/qwindowsxpstyle.cpp
index d36011c..a5e9c19 100644
--- a/src/gui/styles/qwindowsxpstyle.cpp
+++ b/src/gui/styles/qwindowsxpstyle.cpp
@@ -2154,7 +2154,7 @@ void QWindowsXPStyle::drawControl(ControlElement element, const QStyleOption *op
                 p->setPen(menuitem->palette.text().color());
                 p->setBrush(Qt::NoBrush);
                 if (checked)
-                    p->drawRect(vIconRect.adjusted(-1, -2, 1, 1));
+                    p->drawRect(vIconRect.adjusted(-1, -1, 0, 0));
                 p->drawPixmap(vIconRect.topLeft(), pixmap);
 
             // draw checkmark -------------------------------------------------
-- 
1.7.0.2.msysgit.0

